workshop/library/ui/controllers/sidebarInterface.lua

```diff
@@ -35,15 +35,24 @@ return {
             position = guiCoord(0, 0, 0.24, 0),
             backgroundColour = globals.defaultColours.secondary,
         })
+<<<<<<< HEAD
```

Comment:
This looks like something went wrong when resolving conflicts

Reply:
Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.